charts/sequencer/templates/service.yaml

```diff
@@ -34,3 +34,17 @@ spec:
   type: NodePort
   {{- end }}
 ---
+{{- if .Values.serviceMonitor.enabled }} 
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ .Values.config.moniker }}-sequencer-metrics
+  namespace: {{ .Values.global.namespace }}
+spec:
+  selector:
+    app: {{ .Values.config.moniker }}-sequencer
```

Comment:
Do we need to update this and other areas to use the new helpers name and labels?

Reply:
we dont have to do it this way at all either, was just a fast way to ensure service monitor selectors picked up the right app name